pkgs/by-name/sm/smc-fuzzer/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  swiftPackages,
+  fetchFromGitHub,
+  stdenv,
+}:
+
+# This isn’t just useful for fuzzing; it can also be used to explicitly
+# communicate with the SMC.  That makes it useful in system management,
+# e.g. querying / controlling charge status of an Apple laptop.
+stdenv.mkDerivation {
+  pname = "smc-fuzzer";
+  version = "20201223";
```

```suggestion
  version = "0-unstable-2020-12-23";
```
From the [Package Naming Conventions](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming):

> * If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning / tagging schema - then the latest upstream version in the above schema should be 0.
>
>   Example: Given a project that has no tags / released versions at all, or applies versionless tags like latest or YYYY-MM-DD-Build, a commit authored on March 15, 2022 would have version = "0-unstable-2022-03-15".

---

Thanks I was looking for this but I couldn't find it anywhere. I ended up https://nixos.org/manual/nixpkgs/stable/#sec-using-stdenv here but gave up in the end.

I'll change it 👍